ReactiveCocoa/Swift/Action.swift

```diff
@@ -140,34 +139,34 @@ public protocol ActionType {
 	/// with the given input, then forward the results upon the produced Signal.
 	///
 	/// If the action is disabled when the returned SignalProducer is started,
-	/// the produced signal will send `ActionError.NotEnabled`, and nothing will
+	/// the produced signal will send `ActionError.disabled`, and nothing will
 	/// be sent upon `values` or `errors` for that particular signal.
-	func apply(input: Input) -> SignalProducer<Output, ActionError<Error>>
+	func apply(_ input: Input) -> SignalProducer<Output, ActionError<Error>>
 }
 
-extension Action: ActionType {
+extension Action: ActionProtocol {
 	public var action: Action {
 		return self
 	}
 }
 
 /// The type of error that can occur from Action.apply, where `Error` is the type of
 /// error that can be generated by the specific Action instance.
-public enum ActionError<Error: ErrorType>: ErrorType {
+public enum ActionError<Error: ErrorProtocol>: ErrorProtocol {
 	/// The producer returned from apply() was started while the Action was
 	/// disabled.
-	case NotEnabled
+	case disabled
 
 	/// The producer returned from apply() sent the given error.
-	case ProducerError(Error)
+	case producerError(Error)
```

Comment:
Can we do what Result [is doing](https://github.com/antitypical/Result/blob/2fe88d1d41615ed060489e8cd06fc1b3e8f0ca53/Result/Result.swift#L179-189) and add `@unavailable` static funcs with the old names?